ord_schema/message_helpers.py

```diff
@@ -316,17 +322,43 @@ def load_message(filename, message_type, input_format):
         ValueError: if the message cannot be parsed, or if `input_format` is not
             supported.
     """
-    if input_format == 'binary':
+    input_format = MessageFormats(input_format)
```

Comment:
This is an interesting way to check allowable values -- is this a common use of `enum.Enum`? I've never used `enum`.

Reply:
I was very surprised that I couldn't find anything in the `enum` documentation about testing for bad values; I found that this worked well and had the nice side effect of returning the Enum member (since `MessageFormats.BINARY != 'binary'`, but `MessageFormats.BINARY == MessageFormats('binary')`).

I like `enum` mostly for constants that are used in multiple places. Also this avoid needing to add a separate "unsupported type" error since `MessageFormats` will raise an exception if you pass in an invalid type.